google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/test/java/com/google/cloud/logging/logback/LoggingAppenderTest.java

```diff
@@ -246,4 +250,28 @@ private LoggingEvent createLoggingEvent(Level level, long timestamp) {
     loggingEvent.setTimeStamp(timestamp);
     return loggingEvent;
   }
+
+  @Test
+  public void testMdcValuesAreConvertedToLabelsWithPassingNullValues() {
+    MDC.put("mdc1", "value1");
```

Comment:
Not really sure about this but can we not something like this? 

```java
LogEntry logEntry =
        LogEntry.newBuilder(StringPayload.of("this is a test"))
            .setTimestamp(100000)
            .setSeverity(Severity.ERROR)
            .setLabels(
                new ImmutableMap.Builder<String, String>()
                    .put("mdc1", "value1")
                    .put("mdc2", null)
                    .put("mdc3", "value3")
                    .build())
            .build();
```

Reply:
we can but to generate case of NPE  MDC.put() is used